library/spdm_common_lib/libspdm_com_context_data.c

```diff
@@ -2250,14 +2243,6 @@ void libspdm_reset_context(void *spdm_context)
     context->last_spdm_request_size = 0;
     context->encap_context.certificate_chain_buffer.max_buffer_size =
         sizeof(context->encap_context.certificate_chain_buffer.buffer);
-
-    #if LIBSPDM_ENABLE_MSG_LOG
-    context->msg_log.buffer = NULL;
```

Comment:
We can keep the buffer_size and status though, right?

Reply:
Nothing in the message logger gets reset by `libspdm_reset_context`.